.github/workflows/CI.yml

```diff
@@ -61,6 +64,9 @@ jobs:
         run: |
           sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
           sudo apt-get install -yqq libgeos-dev valgrind
+      - name: Set Maintainer Mode
+        if: ${{ github.event.pull_request }}
+        run: export MAINTAINER_MODE=1
```

Comment:
Why the need to set that in memcheck as well?

Reply:
It's not strictly necessary, but I figured we should add it here as well since it still compiles everything and runs the tests while also checking for leaks.

Reply:
So the issue I have with that is that memcheck is already slower to run. However, since tests on mac are even slower, and we are still around the 1mn mark, LGTM :)

Reply:
I think it's fine to remove it from that then. As long as we have it on the main test runner we'll be good.